testsuite/tests/tool-ocamlc-compat32/is_64bit.ml

```diff
@@ -1,2 +1,2 @@
 let () =
-  if Sys.word_size = 64 then exit 0 else exit 1
+  if Sys.int_size >= 63 then exit 0 else exit 1
```

Comment:
This change isn't necessary - the removal of the `xargs` call is sufficient for Windows. Personally the original is fine, but if it's going to be an inequality, shouldn't it be `> 32`?

However, I also think this is a bit too heavyweight - could you just grep the output of `$(OCAMLC) -config`?